services/src/main/java/edu/unc/lib/dl/cdr/services/processing/SetAsPrimaryObjectService.java

```diff
@@ -89,7 +88,7 @@ public void setRepositoryObjectLoader(RepositoryObjectLoader repoObjLoader) {
     }
 
     /**
-     * @param operatonsMessageSender the message sender to set
+     * @param indexingMessageSender the message sender to set
```

Comment:
need to switch this javadoc back to `operatonsMessageSender`

Reply:
done